### PR TITLE
Fix crash on SelectAll in header

### PR DESCRIPTION
**Pull request by amcasey**
Before the first prompt, there is no preceding prompt.

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @cston @ManishJayaswal @KevinH-MS @tmat @genlu

File: src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs [hunk 1]
```diff
@@ -726,6 +726,17 @@ public void CancelMultiLineInput()
...
 
+        [Fact]
+        public void SelectAllInHeader()
+        {
+            Window.WriteLine("Header");
+            Window.FlushOutput();
+            Assert.Equal("Header\r\n> ", Window.TextView.TextBuffer.CurrentSnapshot.GetText());
+
+            Window.TextView.Caret.MoveTo(new SnapshotPoint(Window.TextView.TextBuffer.CurrentSnapshot, 1));
+            Window.Operations.SelectAll(); // Used to throw.
```
Comment (cston) [inline src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs]: What should be selected at this point? Perhaps add Assert for selected range or content.

**Comment by cston** [comment]
LGTM

**Comment by KevinH-MS** [comment]
:+1:

File: src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs (hunk 1 above)
Comment (amcasey) [inline src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs]: Since we're not in a submission, all text is selected.  I've added an assert.

**Pull request merged**